OptimizelySDK/AudienceConditions/SemanticVersion.cs

```diff
@@ -54,7 +64,17 @@ public static bool IsPreRelease(this string semanticVersion)
         /// <returns>True if Semantic version contains '+', else false</returns>
         public static bool IsBuild(this string semanticVersion)
         {
-            return semanticVersion.Contains(BuildSeparator.ToString());
+            var buildIndex = semanticVersion.IndexOf(BuildSeparator.ToString());
+            var preReleaseIndex = semanticVersion.IndexOf(PreReleaseSeparator.ToString());
+            if (preReleaseIndex < 0)
+            {
+                return buildIndex > 0;
+            }
+            else if (buildIndex < 0)
+            {
+                return false;
+            }
+            return buildIndex < preReleaseIndex;
```

Comment:
A unified `IsPrereleaseOrBuild` method may be slightly more performant. I'd recommend going down that path.